### DISCUSSION TITLE
infura and alchemy

### DISCUSSION THREAD

**Discussion by gunomnifluencer** [smartcontractkit/full-blockchain-solidity-course-py#392] [Q&A] [answered]
Hello

Can someone explain to me about **infura** and **alchemy**?

what are both used for actually?

Thanks

**Comment by cromewar** [comment, marked as answer]
Hello @gunomnifluencer the question is tricky as both infura and alchemy provides a lot of services, so I'm going to point you to detailed explanations:

- https://decrypt.co/resources/what-is-infura
- https://www.youtube.com/watch?v=HFs5XA2Xmac
- https://www.youtube.com/watch?v=8X7SkLM2sbA

If you want a one line answer: well they help you to connect, access to https protocols and manage your smart contract deployed on most of the blockchains. 

Best regards, Cromewar

**Reply by gunomnifluencer** [reply]
Hello,

Thanks for answer,

do I still need to use infura to deploy eth to **mainet** with real money?

**Reply by cromewar** [reply]
Depends on you, brownie uses infura by default but you are free to add any other service you want.

**Reply by gunomnifluencer** [reply]
Hello, 

Thanks for answer, 

Is it possible to deploy eth to mainnet without **service**?

**Reply by cromewar** [reply]
Depends on the technologies you are working with I strongly suggest you to check this link of official ethereum docs: https://ethereum.org/en/developers/docs/smart-contracts/deploying/

Best regards, Cromewar.
